desktop/components/marketing_signup_modal/index.coffee

```diff
@@ -54,7 +54,8 @@ module.exports = class MarketingSignupModal extends Backbone.View
 
     # This is just for the Seattle Art Fair campaign and should be removed after
     # the fair closes.
-    slug = 'ca3' if !slug and sd.CURRENT_PATH is '/seattle-art-fair-2017'
```

Comment:
Dunno if we plan to use this code at all in the future, but if so it would be good to put that string into a config var too so we could easily toggle this off without having to deploy.